provider/resource_kpi_threshold_template.go

```diff
@@ -36,15 +37,17 @@ func NewResourceKpiThresholdTemplate() resource.Resource {
 }
 
 type modelKpiThresholdTemplate struct {
-	ID                                 types.String                             `tfsdk:"id" json:"_key"`
-	Title                              types.String                             `tfsdk:"title" json:"title"`
-	Description                        types.String                             `tfsdk:"description" json:"description"`
-	AdaptiveThresholdingTrainingWindow types.String                             `tfsdk:"adaptive_thresholding_training_window" json:"adaptive_thresholding_training_window"`
-	TimeVariateThresholds              types.Bool                               `tfsdk:"time_variate_thresholds" json:"time_variate_thresholds"`
-	TimeVariateThresholdsSpecification *TimeVariateThresholdsSpecificationModel `tfsdk:"time_variate_thresholds_specification"`
-	AdaptiveThresholdsIsEnabled        types.Bool                               `tfsdk:"adaptive_thresholds_is_enabled" json:"adaptive_thresholds_is_enabled"`
-	SecGrp                             types.String                             `tfsdk:"sec_grp" json:"sec_grp"`
-
+	ID                                              types.String                             `tfsdk:"id" json:"_key"`
+	Title                                           types.String                             `tfsdk:"title" json:"title"`
+	Description                                     types.String                             `tfsdk:"description" json:"description"`
+	AdaptiveThresholdingTrainingWindow              types.String                             `tfsdk:"adaptive_thresholding_training_window" json:"adaptive_thresholding_training_window"`
+	TimeVariateThresholds                           types.Bool                               `tfsdk:"time_variate_thresholds" json:"time_variate_thresholds"`
+	TimeVariateThresholdsSpecification              *TimeVariateThresholdsSpecificationModel `tfsdk:"time_variate_thresholds_specification"`
+	AdaptiveThresholdsIsEnabled                     types.Bool                               `tfsdk:"adaptive_thresholds_is_enabled" json:"adaptive_thresholds_is_enabled"`
+	AdaptiveThresholdingOutlierExclusionAlgo        types.String                             `tfsdk:"adaptive_thresholding_outlier_exclusion_algo"`
+	AdaptiveThresholdingOutlierExclusionSensitivity types.Float64                            `tfsdk:"adaptive_thresholding_outlier_exclusion_sensitivity"`
```

Comment:
if you will do 
`tfsdk:"adaptive_thresholding_outlier_exclusion_algo" json:"adaptive_thresholding_outlier_exclusion_algo"` it will be parsed on unmarshalBasicTypesByTag :)

Reply:
I know, but i don't always want to parse them though. I'm only parsing them when `aggregate_outlier_detection_enabled != true` (https://github.com/TiVo/terraform-provider-splunk-itsi/pull/527/files#diff-f3d4e28cd6a9b8d66f6d28644fbc61f325f5ac956fb8eebd46d4923e98e5cbeaR652-R666), otherwise I leave them as Null values.

Reply:
I saw, thank you, I would just reset this after (just because less code), smth like 
```
if (!aggregate_outlier_detection_enabled) {
   adaptive_thresholding_outlier_exclusion_algo = ""
}
```

But it was just in case, you didn't notice, I do not require any changes, looks good!